scripts/test-docker

```diff
@@ -13,14 +13,14 @@ while true; do
   sleep 5
 done
 
-test_docker_versions["ubuntu:18.04"]="^19.03.15$|^20.10"
-test_docker_versions["ubuntu:20.04"]="^^19.03.15$|^20.10"
-test_docker_versions["ubuntu:22.04"]="^20.10.1[3-9]|^20.10.2[0-9]"
-test_docker_versions["centos:centos7"]="^19.03.15$|^20.10"
-test_docker_versions["centos:centos8"]="^20.10.[7-9]|^20.10.1[0-9]|^20.10.2[0-9]"
-test_docker_versions["oraclelinux:7"]="^19.03.15$|^20.10"
-test_docker_versions["oraclelinux:8"]="^20.10"
-test_docker_versions["rockylinux:8"]="^20.10.[8-9]|^20.10.1[0-9]|^20.10.2[0-9]"
+test_docker_versions["ubuntu:18.04"]="^19.03.15$|^20.10|^23.0.([0-9]|10)"
```

Comment:
Looks like this only includes 0 through 10. Why not simply go with below, same as we do for 20.10:
```suggestion
test_docker_versions["ubuntu:18.04"]="^19.03.15$|^20.10|^23.0"
```

Reply:
@snasovich I updated to include all future 23.0 docker releases. Can you please review it again?